src/types/forgotPassword.ts

```diff
@@ -0,0 +1,47 @@
+import { NativeStackNavigationProp } from '@react-navigation/native-stack'
+import { RootStackParamList } from '@/types/navigator'
+
```

Comment:
Quite a few re-usable types in this file, can we define them once and re-use please?